main.py

```diff
@@ -19,6 +19,7 @@
 import json
 import time
 import threading
+import camera as cam
```

Comment:
Let me mention this here, although this is more a file structure thing. Since you have started splitting the code into several modules, it would make sense to move all modules into a Python package. Essentially just create a subfolder `cameracontrol` (we should still talk about the final name) and moce:

- `camera.py` → `cameracontrol/camera.py`
- `main.py` → `cameracontrol/__main__.py`

You can call a package directly with:

```
python -m cameracontrol
```

This import would also become something like:

```py
from cameracontrol.camera import Camera
```

…assuming that you just want to import the camera class.